### DISCUSSION TITLE
<Random Skin> isn't named correctly (osu.Game.Database.RealmLiveUnmanaged'1)

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#16138] [Q&A] [answered]
I think that this happened with the new osu!lazer update.

![osu_2021-12-17_15-31-42](https://user-images.githubusercontent.com/74088315/146610394-6e77aab0-0397-45b3-a061-72d16f8d1b0e.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16141
